mountpoint-s3/src/fs.rs

```diff
@@ -1,3 +1,31 @@
+//! File system types and operations, not relying on our FUSE library.
```

Comment:
general nit on this comment: we've been roughly trying to wrap at at 120 chars wherever we can (which is what our rustfmt config is set to, but it doesn't affect comments).